evm/README.md

```diff
@@ -244,3 +244,8 @@ bash sh/configure_wormhole_ntt.sh -n NETWORK_TYPE -c CHAIN_NAME -k PRIVATE_KEY
 -n testnet, mainnet
 -c avalanche, ethereum, sepolia
 ```
+
+#### Additional Notes
+Tokens powered by NTT require the `burn` method to be present. This method is not present in the standard ERC20 interface, but is found in the `ERC20Burnable` interface.
```

Comment:
This is only true when the contract is in `burn` mode. NTT can support existing ERC20's without `burn` by using `locking` mode.